### ISSUE TITLE
Compile problems

### ISSUE THREAD

**Issue by bphermansson** [ImpulseAdventure/GUIslice#51]
What can be the problem here?

```
/media/nasFiler/Programmering/Arduino/sketchbook/libraries/GUIslice/src/GUIslice_drv_tft_espi.cpp: In function 'bool gslc_DrvGetTouch(gslc_tsGui*, int16_t*, int16_t*, uint16_t*)':
/media/nasFiler/Programmering/Arduino/sketchbook/libraries/GUIslice/src/GUIslice_drv_tft_espi.cpp:822:21: error: 'class TFT_eSPI' has no member named 'getTouch'
   bPressed = m_disp.getTouch(&nX,&nY);
                     ^
exit status 1
Error compiling for board WeMos D1 mini Lite.
```
I have a ST7735-baseddisplay without touch.

**Comment by ImpulseAdventure** [comment]
Hi @bphermansson --

It sounds like you don't require touch support yet GUIslice may have been configured to enable it for the TFT_eSPI library. Can you confirm that `GUIslice_config_ard.h` has enabled `#define DRV_TOUCH_NONE` and commented out `#define DRV_TOUCH_TFT_ESPI`?

Thanks!

**Comment by bphermansson** [comment]
Yes it's set that way. 

```
  #define DRV_TOUCH_NONE            // No touchscreen support
  //  #define DRV_TOUCH_ADA_STMPE610    // Adafruit STMPE610 touch driver
  //  #define DRV_TOUCH_ADA_FT6206      // Adafruit FT6206 touch driver
  //  #define DRV_TOUCH_ADA_SIMPLE      // Adafruit Touchscreen
  //  #define DRV_TOUCH_TFT_ESPI        // TFT_eSPI integrated XPT2046 touch driver
```

**Comment by ImpulseAdventure** [comment]
Thanks Patrik for providing your config. Given the line number reported in the error it seems like you might be running an older version (before May 21) of the `GUIslice_drv_tft_espi.cpp` code which had an issue involving `DRV_TOUCH_NONE` (addressed in 6e8fa4800cdc6d8292e986ce2d727d11435292a5).

If this might be the case, could you re-test with the latest version?

thanks!

**Comment by bphermansson** [comment]
Indeed. I used the version from Arduinos library manager which was 0.10.0 I think released in march. I now tried to replace it with the git-version, "git clone..." and now it compiles. 
Maybe time to upgrade the library version?
